[IMP] helpdesk: update closing tickets (#7888)
odoo/documentation: content/applications/services/helpdesk/advanced/close_tickets.rst

=============
Close tickets
=============

Once work has been completed on a *Helpdesk* ticket in Odoo, there are several ways it can be

> closed. In addition to manually closing solved tickets, automatically closing inactive tickets keeps
> the pipeline up to date. At the same time, allowing customers to close their own tickets minimizes
> confusion around whether an issue is considered solved or not. This results in increased operational
> capacity for support teams, and higher customer satisfaction.

Reviewer: ```suggestion
closed.

In addition to manually closing solved tickets, automatically closing inactive tickets keeps the
pipeline up-to-date. At the same time, allowing customers to close their own tickets minimizes
confusion around whether an issue is considered solved or not. This results in increased operational
capacity for support teams, and higher customer satisfaction.
```
OPTIONAL: broke up an otherwise slightly "chunky" paragraph

=============
Close tickets
=============

Once work has been completed on a *Helpdesk* ticket in Odoo, there are several ways it can be

> closed. Manually closing solved tickets keeps the pipeline up to date, while automatically closing

Reviewer 2: Just curious- how does not automatically closing tickets make a blocking issue? Wouldn't this just have open ticket numbers remain high? I thought a blocking issue was something stopping something form happening- so I don't see this as blocking, but I don't use Helpdesk, so I could definitely be wrong!

capacity for support teams, and higher customer satisfaction.

Manually close solved tickets
=============================

As work on a ticket progresses, it is moved along to the next stage in the pipeline. Once the issue
is solved, the ticket is moved to a *folded* stage. This marks the ticket as *closed*.

To fold a stage, navigate to the :menuselection:`Helpdesk` app dashboard, and click on a team to
open the to reveal that team's pipeline on a separate page.

From the pipeline page, hover over a stage's heading, and click the :guilabel:`⚙️ (gear)` icon that
appears in the top-right corner of that stage's Kanban column.

.. image:: close_tickets/closing-edit-stage-gear.png
   :align: center
   :alt: View of stage on Helpdesk pipeline with emphasis on gear icon and edit stage option.

From the menu that appears, select :guilabel:`Edit`. This opens the stage's settings in a pop-up
window.

In the pop-up window, check the box labeled, :guilabel:`Folded in Kanban`, towards the top of the
window. Then, click :guilabel:`Save & Close` to confirm the changes.

Tickets that reach this stage are now considered: *closed*.

>    .. image:: close_tickets/closing-folded-setting.png
>       :align: center
>       :alt: Stage settings page.

Reviewer 3: No need for this image to be indented :) scoot it back please
```suggestion
.. image:: close_tickets/closing-folded-setting.png
   :align: center
   :alt: Stage settings page.
```

is solved, the ticket is moved to a *folded* stage. This marks the ticket as *closed*.

To fold a stage, navigate to the :menuselection:`Helpdesk` dashboard and click on a team to open the
pipeline. Hover over a stage's heading, and then click the :guilabel:`⚙️ (gear)` icon that appears
in the top-right corner of that stage's Kanban column.

.. image:: close_tickets/closing-edit-stage-gear.png
   :align: center
   :alt: View of stage on Helpdesk pipeline with emphasis on gear icon and edit stage option.

.. warning::
   Clicking the :guilabel:`⚙️ (gear)` icon also displays the option to :guilabel:`Fold` the stage.
   This setting folds the stage *temporarily* to simplify the Kanban view. This does **not** close
   the tickets in this stage. It also does **not** permanently fold the stage. If a stage needs to

>    be folded so the tickets can be marked as closed, continue following the steps below.

Reviewer 2: Just so this is not confusing, I would put this warning after the next part. So complete the steps to make it a folded stage, then have the warning about the temporary folding.

is solved, the ticket is moved to a *folded* stage. This marks the ticket as *closed*.

> To fold a stage, navigate to the :menuselection:`Helpdesk` dashboard and click on a team to open the

Reviewer: ```suggestion
To fold a stage, navigate to the :menuselection:`Helpdesk` app dashboard, and click on a team to open the
```

> pipeline. Hover over a stage's heading, and then click the :guilabel:`⚙️ (gear)` icon that appears
> in the top-right corner of that stage's Kanban column.

Reviewer: ```suggestion
to reveal that team's pipeline on a separate page. 

From the pipeline page, hover over a stage's heading, and click the :guilabel:`⚙️ (gear)` icon that
appears in the top-right corner of that stage's Kanban column.
```

.. image:: close_tickets/closing-edit-stage-gear.png
   :align: center
   :alt: View of stage on Helpdesk pipeline with emphasis on gear icon and edit stage option.

> From the menu that appears, select :guilabel:`Edit`. This opens the stage's settings. Check the box
> labeled :guilabel:`Folded in Kanban` towards the top of the window, and then :guilabel:`Save &
> Close` to confirm the changes. Tickets that reach this stage are now considered as *closed*.

Reviewer: ```suggestion
From the menu that appears, select :guilabel:`Edit`. This opens the stage's settings in a pop-up
window.

In the pop-up window, check the box labeled, :guilabel:`Folded in Kanban`, towards the top of the
window. Then, click :guilabel:`Save & Close` to confirm the changes.

Tickets that reach this stage are now considered: *closed*.
```

   .. image:: close_tickets/closing-folded-setting.png
      :align: center
      :alt: Stage settings page.

.. warning::
   Clicking the :guilabel:`⚙️ (gear)` icon also displays the option to :guilabel:`Fold` the stage.
   This setting folds the stage *temporarily* to simplify the Kanban view. This does **not** close
   the tickets in this stage. It also does **not** permanently fold the stage. If a stage needs to

>    be folded so the tickets can be marked as closed, the :guilabel:`Folded in Kanban` checkbox

Reviewer: ```suggestion
   be folded, so the tickets can be marked as `closed`, the :guilabel:`Folded in Kanban` checkbox
```

   **must** be checked on the stage's settings.

Automatically close inactive tickets
====================================

Tickets that are inactive for a set period of time can be automatically closed. At that point, they
are moved to a folded stage.

Go to the team's settings page by going to :menuselection:`Helpdesk app --> Configuration -->
Helpdesk Teams`. Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.

> If one of the team's stages is set to be folded in the Kanban view, the folded stage is the default
> selection in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the
> folded stage that occurs first in the pipeline is the default. If no stage is folded, the default
> selection is the last stage in the pipeline.

Reviewer: ```suggestion
After ticking the box for :guilabel:`Automatic Closing`, three new fields appear beneath:

- :guilabel:`Move to Stage`
- :guilabel:`After (#) days of inactivity`
- :guilabel:`In Stages`

If one of the team's stages is set to be folded in the Kanban view, the folded stage is the default
selection in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the
folded stage that occurs first in the pipeline is the default. If no stage is folded, the default
selection is the last stage in the pipeline.
```
just for some more detail re: the fields that appear after ticking the box for that feature

====================================

Tickets that are inactive for a set period of time can be automatically closed. At that point, they
are moved to a folded stage.

Go to the team's settings page by going to :menuselection:`Helpdesk app --> Configuration -->
Helpdesk Teams`. Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.

If one of the team's stages is set to be folded in the Kanban view, the folded stage is the default
selection in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the

> stage that occurs first in the pipeline is the default. If no stage is folded, the default selection

Reviewer 2: ```suggestion
folded stage that occurs first in the pipeline is the default. If no stage is folded, the default selection
```

is the last stage in the pipeline.

> The :guilabel:`After days of inactivity` field defaults to `7`, but can be adjusted if necessary.

Reviewer 2: ```suggestion
The :guilabel:`After (#) days of inactivity` field defaults to `7`, but can be adjusted if necessary.
```

Not sure if that's the best way, but I just wanted to note that there is a value between the text up there.  Maybe "After X days"?

   - `Customer Feedback`
   - `Closed`

>    Tickets may linger in the :guilabel:`Customer Feedback stage`, because once an issue is solved,
>    customers may not respond immediately. At that point, the tickets can be closed automatically.

Reviewer 2: ```suggestion
   Tickets may linger in the :guilabel:`Customer Feedback` stage, because once an issue is solved,
   customers may not respond immediately. At that point, the tickets can be closed automatically.
```

   - `Customer Feedback`
   - `Closed`

   Tickets may linger in the :guilabel:`Customer Feedback` stage, because once an issue is solved,
   customers may not respond immediately. At that point, the tickets can be closed automatically.

   Tickets in the :guilabel:`New` and :guilabel:`In Progress` stages could remain inactive due to
   assignment or workload issues. The support team may be looking into the issue even if they are
   not updating the ticket directly. Closing these tickets automatically would result in issues
   going unsolved.

   Therefore, the :guilabel:`Automatic Closing` settings for this team would be configured as

>    below\:\

Reviewer: is there a reason behind the slashes before and after the ":" here -- it looks like it renders fine when I run 'make fast.' I just wasn't super sure if this was intentional or not -- either way, just figured it was worth bringing to your attention. 😄
